src/query/storage/m3/storage.go

```diff
@@ -51,8 +59,16 @@ type m3storage struct {
 
 // NewStorage creates a new local m3storage instance.
 // TODO: Consider combining readWorkerPool and writeWorkerPool
-func NewStorage(clusters Clusters, workerPool pool.ObjectPool, writeWorkerPool xsync.PooledWorkerPool) Storage {
-	return &m3storage{clusters: clusters, readWorkerPool: workerPool, writeWorkerPool: writeWorkerPool}
+func NewStorage(
```

Comment:
i thought artem had removed the TODO above in another PR. mind rebasing on master once?

Reply:
Sure thing.

Reply:
sigh must have been skipped in the last review, mind deleting it please.